scripts/create_users.go

```diff
@@ -47,3 +49,25 @@ func CreateUsers(config *serializers.Config, logger *zap.Logger) error {
 
 	return nil
 }
+
+func GetOrCreateUser(client *model.Client4, userDetails serializers.UsersConfiguration) (*model.User, error) {
```

Comment:
```suggestion
func GetOrCreateUser(client *model.Client4, userDetails *serializers.UsersConfiguration) (*model.User, error) {
```
Use pointers everywhere.